website/src/components/Chat/ChatSection.tsx

```diff
@@ -1,37 +1,37 @@
 import { Card, CardBody, Divider } from "@chakra-ui/react";
+import dynamic from "next/dynamic";
 import { FormProvider, useForm } from "react-hook-form";
+import { useCacheConfig } from "src/hooks/chat/useCacheConfig";
 import { ChatConfigForm } from "src/types/Chat";
 
-import { ChatConfigSummary } from "./ChatConfigSummary";
+import { ChatConfigSaver } from "./ChatConfigSaver";
 import { useChatContext } from "./ChatContext";
 import { ChatConversation } from "./ChatConversation";
 import { InferencePoweredBy } from "./InferencePoweredBy";
 
+const ChatConfigSummary = dynamic(() => import("./ChatConfigSummary"), { ssr: false });
```

Comment:
why is this necessary?

Reply:
The local storage cause a hydration error. I'm can't find a way to make it gratefully hydrate. It's the easiest solution for now. The ChatConfigSummary will be removed on the next UI refactor I think.